web/src/filter.ts

```diff
@@ -1100,18 +1118,19 @@ export class Filter {
     }
 
     filter_with_new_params(params: NarrowTerm): Filter {
+        const new_params = this.fix_terms([params])[0];
```

Comment:
This change could probably be a prep commit, since it deserves its own commit message about how we need to use canonical version of terms to do the adjustments correctly.

Reply:
Separated this out into a separate prep commit.